src/nodeInfo.tsx

```diff
@@ -5,7 +5,7 @@ import { useSubstrate } from './substrate-lib';
 
 function Main (props) {
   const { api, socket } = useSubstrate();
-  const [nodeInfo, setNodeInfo] = useState({});
+  const [nodeInfo, setNodeInfo] = useState<NodeDTO>((new NodeDTO()));
```

Comment:
Let's remove this and let the future PR handle it.

Reply:
Doen